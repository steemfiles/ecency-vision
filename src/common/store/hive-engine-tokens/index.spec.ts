import reducer, {initialState } from "./index";
let state = {};

it("1- default state", () => {
    expect(state).toMatchSnapshot();
});
