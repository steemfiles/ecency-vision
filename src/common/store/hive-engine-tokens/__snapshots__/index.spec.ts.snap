// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1- default state 1`] = `Object {}`;
